src/analysis/retail/paladin/protection/modules/talents/FirstAvenger.tsx

```diff
@@ -21,16 +21,16 @@ class FirstAvenger extends Analyzer {
 
   constructor(options: Options) {
     super(options);
-    this.active = this.selectedCombatant.hasTalent(SPELLS.FIRST_AVENGER_TALENT);
+    this.active = this.selectedCombatant.hasTalent(TALENTS.FERREN_MARCUSS_FERVOR_TALENT);
```

Comment:
this is the wrong talent for this. the old talent First Avenger got split into two: Soaring Shield + Bulwark of Order. the extra damage tracked here is for the extra bounces from Soaring Shield.

Reply:
I swapped these to be more specific to soaring shield talent